### PR TITLE
docs: community content

### PR DESCRIPTION
**Pull request by vinckr**
Added the docker compose examples and articles from @radekg. 
Thanks a ton for the writeup and examples 🎉

### CODE REVIEW

**Comment by vinckr** [comment]
Oh, very nice; 
I will add that repo as well. 
Looking forward to a writeup, have been enjoying reading through your articles, even the ones not about Ory 🙂

File: docs/docs/ecosystem/community.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ This is a list of those.
 
 - Open source software
 
+  - [Reference ORY Docker Compose Setup ](https://github.com/radekg/ory-reference-compose) + [Article](https://gruchalski.com/posts/2021-04-10-ory-reference-docker-compose-and-thoughts-on-the-platform/)
```
Comment (aeneasr) [inline docs/docs/ecosystem/community.md:30]:
```suggestion
  - [Reference ORY Docker Compose Setup](https://github.com/radekg/ory-reference-compose) + [Article](https://gruchalski.com/posts/2021-04-10-ory-reference-docker-compose-and-thoughts-on-the-platform/)
```

**Comment by vinckr** [comment]
@radekg 

Sorry for pinging you like this, but I figured it would be a good way to ask you something. 

Since we like your blogpost a lot, we would love to repost it on our company blog (ory.sh/blog).
So you would be a guest author.
We would include a paragraph at then end, where you can promote some of your website and work as well.
Additionally I would rephrase parts of your post, keeping the original meaning intact, so we dont get penalized by SEO. 

Let me know if that is something you would enjoy (or not 🙂 ).  


Ps: Also let me know if you would like to receive a Ory shirt as a small token of appreciation.
(regardless of you being a guest author or not)

**Comment by radekg** [comment]
Hey @vinckr, sure, shall we discuss details offline? Your email is in the profile, if that's okay with you, I'll drop you a line?

**Comment by vinckr** [comment]
@radekg Yes please! 
Feel free to email me !
